UnitTests/MPRoktTests.m

```diff
@@ -155,4 +164,74 @@ - (void)testSelectPlacementsExpandedWithNilParameters {
     OCMVerifyAll(mockContainer);
 }
 
+- (void)testSelectPlacementsSimpleWithMapping {
```

Comment:
This test stubs the `getRoktPlacementAttributes` call.  

Can you add a test for `getRoktPlacementAttributes` itself, using a config like the one you put in the chat.  passing in "[{\"jsmap\":null,\"map\":\"f.name\",\"maptype\":\"UserAttributeClass.Name\",\"value\":\"firstname\"},{\"jsmap\":null,\"map\":\"zip\",\"maptype\":\"UserAttributeClass.Name\",\"value\":\"billingzipcode\"},{\"jsmap\":null,\"map\":\"l.name\",\"maptype\":\"UserAttributeClass.Name\",\"value\":\"lastname\"}]"

and expecting what you have in line 168 back?

(note that i copied and pasted what you had in #temp-rokt-client but updated the mapping from `f.name` --> `firstname` instead of email